firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteRemoteDocumentCache.java

```diff
@@ -135,7 +143,7 @@ public Map<DocumentKey, MutableDocument> getAll(Iterable<DocumentKey> documentKe
     while (longQuery.hasMoreSubqueries()) {
       longQuery
           .performNextSubquery()
-          .forEach(row -> processRowInBackground(backgroundQueue, results, row));
+          .forEach(row -> processRowInBackground(backgroundQueue, results, row, null));
```

Comment:
nit: Sometimes it's more readable to write an inline comment whenever it's not clear what a certain parameter is. e.g.
```java
.forEach(row -> processRowInBackground(backgroundQueue, results, row, /*documentFilter*/ null));
```

Reply:
Done.